js/main.js

```diff
@@ -1,12 +1,107 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-console */
 // src https://myrusakov.ru/js-random-numbers.html
 function getRandomInRange(min, max) {
   return Math.floor(Math.random() * (max - min + 1)) + min;
 }
 
-getRandomInRange(1, 10);
-
 // src https://myrusakov.ru/js-random-numbers.html с доработкой из https://stackoverflow.com/questions/17726753/get-a-random-number-between-0-0200-and-0-120-float-numbers
 function getRandomFloat(min, max, floatNuber) {
   return (Math.random() * (max - min) + min).toFixed(floatNuber);
 }
-getRandomFloat(11, 101, 2);
+
+function getRandom(random) {
+  return Math.floor(Math.random() * random);
+}
+
+const getAvatar = () => {
+  const avatarNumber = getRandomInRange(1, 10);
+  if (avatarNumber >= 10) {
+    return `img/avatars/user${avatarNumber}.png`;
+  }
+  return `img/avatars/user${`0${  avatarNumber}`}.png`;
+};
```

Comment:
```suggestion
const getAvatar = () => {
    return `img/avatars/user0${getRandomInRange(1, 9)}.png`;
};
```

и 1/9 в константу вынести